pytorch_lightning/strategies/ddp.py

```
@@ -372,14 +388,19 @@ def broadcast(self, obj: object, src: int = 0) -> object:
 
     def pre_backward(self, closure_loss: torch.Tensor) -> None:
         """Run before precision plugin executes backward."""
+        assert self.model is not None
+        assert self.lightning_module is not None
         if not self.lightning_module.automatic_optimization:
             prepare_for_backward(self.model, closure_loss)
 
-    def model_to_device(self):
+    def model_to_device(self) -> None:
         log.detail(f"{self.__class__.__name__}: moving model to device [{self.root_device}]...")
-        self.model.to(self.root_device)
+        if self.model:
```

I think this should raise an exception if model is None